src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -199,6 +204,13 @@ export default class PhotoField extends React.Component {
     return this.state.errorMessage;
   }
 
+  updateProgress = (progress) => {
+    if (progress === 100) {
```

Comment:
There isn't any progress indication in the designs, but we often provide some communication regarding this for applicants, is this only necessary for other types of files (and can be removed here)?

Reply:
I think we will want to have a progress bar if we need to do uploads and can't save this as base64 in the database. Should have an answer to that this week.

Reply:
Thanks for the helpful context.

Reply:
Here is a proposed progress presentation, if it turns out we decide that we want to communicate this to the applicant, @stacyhlinn @lalexanderson-dsva does this look alright?
![screen shot 2018-01-29 at 10 04 08 pm](https://user-images.githubusercontent.com/16051172/35550748-b410d8e2-0540-11e8-8a99-34af0df41a3a.png)
![screen shot 2018-01-29 at 10 03 36 pm](https://user-images.githubusercontent.com/16051172/35550757-b988cb90-0540-11e8-9d94-08da26679c8e.png)